### PR TITLE
Add node js package and GitHub actions

### PR DESCRIPTION
**Pull request by TheLimeGlass**
### Description
We need to fix GitHub actions running on a node.js branch, also add tests for CI. Ali's pull at https://github.com/SkriptLang/Skript/pull/4319 doesn't have this so it shouldn't conflict.

### CODE REVIEW

**Comment by TheLimeGlass** [comment]
It also prevents GitHub actions from stall idling like this, as it has a task to represent it.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "skript",
+  "version": "1.0.0",
```
Comment (AyhamAl-Ali) [inline package.json:3]: What version would this follow Skript or the website? for the website currently the version is handled [here](https://github.com/SkriptLang/Skript/pull/4319/files#diff-26b330d4ed424c74aa1b75d4ca8936dac30b9c4223f445d39f757c3ec3414453R1)
_In case it doesn't show up from the link_
image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SkriptLang/Skript.git"
+  },
+  "author": "SkriptLang",
+  "license": "ISC",
```
Comment (AyhamAl-Ali) [inline package.json:14]: Shouldn't this follow the repo license? `GPL-3.0 license`

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]

**Review by APickledWalrus — APPROVED** [review]
I think this should be okay (approval from ayham def needed)

**Comment by TheLimeGlass** [comment]
Going to rebase this on the documentation repo

**Pull request closed without merging**